spec/features/user_logs_in_spec.rb

```diff
@@ -58,7 +59,8 @@
   context "as a valid default user" do
     let(:user) { User.create(full_name: "dg",
                               email: "example@example.com",
-                              password: "password") }
+                              password: "password",
+                              display_name: "example name") }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 63, 61 should be on its own line.